content/en/openpnp/FTP/installing-the-feeders/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Installing the Feeders"
+linkTitle: "Installing the Feeders"
+weight: 2
+type: docs
+description: >
+ Getting the feeders ready for automatic picking
+---
+
+The FTP PCB has two different types of components that need to be placed: resistors and LEDs. Each will need its own feeder so that the LumenPnP can pick each type of component. For this guide, you'll use printed tray feeders.
+
+## Attach Feeders
+
+1. Use two M3x10 button head screws and two M3 wingnuts to secure the printed tray feeder through onto the staging plate using holes: C11 and E11.
+  {{< container-image path="images/Feeder-installed-with-screws.jpg" alt="Install the feeder to the staging plate" >}}
+
+2. For each component, cut a strip of component tape off the reel about 125mm long.
+
+3. Slide the Resistors into the left-most tray feeder, and the LEDs the adjacent tray.
+
+4. In OpenPnP, connect to your LumenPnP and home it.
+  {{< container-image path="images/openPnP-shared/Connect-and-home.png" alt="Connect to your LumenPnP and home it" >}}
+
+## Import Board
+<!-- TODO: Download link to Getting Started Board files -->
+5. Download the FTP board design here.
```

Comment:
@sphawes can you figure out where the download link will be living permanently and add it here? Thanks!

Reply:
this will be from the most recent release! i'd link to https://github.com/opulo-inc/lumenpnp/releases/, then have folks download the LumenPnP-PCBs.zip. In that is ftp.zip which has the position file ("ftp-top_pos.csv")